vnext/Microsoft.ReactNative/Version.rc

```diff
@@ -21,15 +21,11 @@
 #define RNW_PKG_VERSION_PATCH 0
 #endif
 
-#ifndef RNW_PKG_VERSION_MAJOR
-#define RNW_PKG_VERSION_MAJOR 1000
+#ifndef RNW_PKG_VERSION_BUILD
```

Comment:
Below still references `RNW_PKG_VERSION_MAJOR`

Would prefer keeping MAJOR in the name, since the version part corresponds to semver major version.

Reply:
Oh I see this is duplicated above.